core/README.md

```diff
@@ -43,12 +43,25 @@ Other properties are also allowed to be provided, but are not described by this
 
 ### General Properties
 
-| Property Name | Data Type | Description |
-| ------------- | --------- | ----------- |
-| id            | string    | **REQUIRED.** A unique identifier for the field. It must be unique within the [Collection](#collection). |
-| collection    | string    | The identifier of the parent collection. |
-
-The collection identifier is usually only needed for merged datasets.
+| Property Name | Data Type      | Description |
+| ------------- | -------------- | ----------- |
+| id            | string         | **REQUIRED.** A unique identifier for the field. It must be unique within the [Collection](#collection). |
+| collection    | string         | The identifier of the parent collection. |
+| category      | array\<string> | A set of categories the field boundary belongs to. |
+
+**collection:** The collection identifier is usually only needed for merged datasets.
+
+**category:** Choose any (unique) combination of the following values:
```

Comment:
TODO: put in a reference to adapt / ag gateway, so people understand fiboa isn't just making this up.
(from fiboa core call 9/27)

Reply:
Link - https://aggateway.org/Portals/1010/WebSite/About%20Us/FIELD%20BOUNDARY%20FLYER%20122123.pdf?ver=2024-01-03-212959-590

It also should be in the adapt standard, likely in data type definition, as an enumeration. So let's also try to link to that - both the flyer providing the overview, and the core spec definition (if it's not there then adapt needs to add it).

Reply:
https://github.com/ADAPT/Standard/issues/97